static/src/stylesheets/module/_headline-list-garnett.scss

```diff
@@ -58,7 +58,18 @@
         margin-left: gs-span(1);
 
         @include mq(mobile) {
-            margin-left: gs-span(1) - $gs-gutter / 2;
+            margin-left: gs-span(1) + $gs-gutter / 2;
+        }
+    }
+
+    .inline-numbers {
+        float: left;
+        margin-top: $gs-baseline / 4;
+
+        svg {
```

Comment:
Minor: You could use the `inline-numbers__svg` instead I think

Reply:
This is fixed by removing the svg entirely.